packages/@vue/cli-plugin-e2e-webdriverio/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@vue/cli-plugin-e2e-webdriverio",
+  "version": "4.3.1",
+  "description": "e2e-webdriverio plugin for vue-cli",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vuejs/vue-cli.git",
+    "directory": "packages/@vue/cli-plugin-e2e-webdriverio"
+  },
+  "keywords": [
+    "vue",
+    "cli",
+    "wdio",
+    "webdriverio",
+    "e2e-webdriverio"
+  ],
+  "author": "Christian Bromann <christian@saucelabs.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/vuejs/vue-cli/issues"
+  },
+  "homepage": "https://github.com/vuejs/vue-cli/tree/dev/packages/@vue/cli-plugin-e2e-webdriverio#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@vue/cli-shared-utils": "^4.3.1",
+    "@wdio/cli": "^6.1.11",
+    "@wdio/local-runner": "^6.1.11",
+    "@wdio/mocha-framework": "^6.1.8",
+    "@wdio/sauce-service": "^6.1.9",
+    "@wdio/spec-reporter": "^6.1.9",
+    "@wdio/sync": "^6.1.8",
+    "chromedriver": "^81.0.0",
```

Comment:
Considering chromedriver updates its major version number quite frequently, could you please make it a peer dependency, like what we did in the nightwatch plugin?

https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-plugin-e2e-nightwatch/package.json
https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-plugin-e2e-nightwatch/prompts.js
https://github.com/vuejs/vue-cli/blob/dev/packages/@vue/cli-plugin-e2e-nightwatch/generator/index.js